src/templates/mkt_nav.html

```diff
@@ -15,6 +24,18 @@
     {{ nav_item(url('new'), _('New'), 'new') }}
     {{ nav_item(url('recommended'), _('Recommended'), 'recommended') }}
     {{ nav_item('#', _('Categories'), 'categories', 'categories') }}
+    <div class="mkt-nav-root--divider"></div>
```

Comment:
This makes the markup `<li>...</li> ... <div></div> ... <li>...</li>`. Not great but I guess it's fine. Should this just be an `<li>` instead though?